source/reviews/standard.txt

```diff
@@ -0,0 +1,90 @@
+======================
+The Standard of Review
+======================
+
+.. _pr-standard:
+
+Purpose
+-------
+
+The primary purpose of a review is to make sure that the overall health of MongoDB's
+corpus is improving over time. All of the tools and processes of review are
+designed to this end.
+
+In order to accomplish this, a series of trade-offs have to be balanced.
+
+First, authors must be able to **make progress** on their tasks. If you never
+submit an improvement to the corpus, then the corpus never improves. Also,
+if a reviewer makes it very difficult for *any* change to go in, then authors
+are disincentivized to make improvements in the future.
+
+On the other hand, it is the duty of the reviewer to make sure that each
+PR is of such a quality that the overall content health of their corpus
+is not decreasing as time goes on. This can be tricky, because often,
+content degrades through small decreases in content health over time,
+especially when a team is under significant time constraints and they feel
+that they have to take shortcuts in order to accomplish their goals.
+
+Also, a reviewer has ownership and responsibility over the code they are
+reviewing. They want to ensure that the corpus stays consistent and maintainable.
+
+The Prime Directive
+-------------------
+
+In general, reviewers should favor approving a PR once it is in a state
+where it definitely improves the overall content health of the area
+being worked on, even if the PR isn't perfect.
```

Comment:
Comment: This "definitely improves the overall content health..." seems restrictive. I think "most likely improves" is more inline with "favor approving a PR"